### DISCUSSION TITLE
Too few arguments to function PHPStan\PhpDocParser\Parser\ConstExprParser::__construct(), 0 passed in /var/www/html/prathin/test/vendor/api-platform/core/src/Metadata/Resource/Factory/PhpDocResourceMetadataCollectionFactory.php on line 62 and exactly 1 expected

### DISCUSSION THREAD

**Discussion by ghost** [api-platform/core#6796] [General]
**_Too few arguments to function PHPStan\PhpDocParser\Parser\ConstExprParser::__construct(), 0 passed in /var/www/html/prathin/test/vendor/api-platform/core/src/Metadata/Resource/Factory/PhpDocResourceMetadataCollectionFactory.php on line 62 and exactly 1 expected_**  I've been facing this issue for the past two days with versions above v3.0. I just downgraded from v3.3 to v3.0. Could you suggest a solution to resolve this in symfony?

**Comment by dunglas** [comment]
Can you try to downgrade PHPStan parser to v1?
Also, this is likely fixed by https://github.com/api-platform/core/pull/6789

**Reply by ghost** [reply]
I'm sorry that when I downgrade, it will affect my other packages.

_**Root composer.json requires phpstan/phpdoc-parser v1.0.0 (exact version match), found phpstan/phpdoc-parser[1.0.0] but these were not loaded, likely because it conflicts with another require.**_

**Reply by soyuka** [reply]
please do not duplicate discussions https://github.com/api-platform/core/discussions/6793
